src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2124,7 +2124,8 @@ console.log(foo + bar) // [!code focus]
 <Code
   code={code}
   lang="js"
-  transformers={[transformerNotationFocus()]} />
+  transformers={[transformerNotationFocus(),[transformerMetaHighlight()]}
```

Comment:
Because `transformerNotationFocus` blurs everything but the line marked with `// [!code focus]`, using it together with `transformerMetaHighlight` makes little sense IMHO.

Or at least the highlighted line should be the same as the focused line.

Reply:
OK! Shall we just delete the NotationFocus transformer then? Would also be OK if you wanted to suggest two that did go together! Just let me know.

Reply:
@sarah11918 sorry it took me some time to circle back to this PR. I see it's merged now.

The example now doesn't work. I has style for a transformer that's not used, and doesn't have any style for the transformer it uses.

The fixed example is that of the original commit: https://github.com/withastro/docs/pull/9054/commits/975a6e1cf9018663a78e9f95adea4316c73ce53b#diff-42959260ea642887b866e8e276a4520c1f112356eb9c15b31e510132a481aed6R2147-R2165